src/ucm/rocm/Makefile.am

```diff
@@ -13,6 +13,9 @@ libucm_rocm_la_CFLAGS   = $(BASE_CFLAGS) $(ROCM_CFLAGS)
 libucm_rocm_la_LIBADD   = ../libucm.la
 libucm_rocm_la_LDFLAGS  = $(UCM_MODULE_LDFLAGS) \
                           $(patsubst %, -Xlinker %, $(ROCM_LDFLAGS)) \
+                          $(patsubst %, -Xlinker %, -L$(ROCM_ROOT)/lib -rpath $(ROCM_ROOT)/hip/lib -rpath $(ROCM_ROOT)/lib) \
+                          $(patsubst %, -Xlinker %, --enable-new-dtags) \
+                          $(patsubst %, -Xlinker %, -rpath $(ROCM_ROOT)/lib64) \
```

Comment:
@srinivamd didn't we decide to remove the code duplication for master?

Reply:
This is to catch up master with v1.8.x. The refactor will be done in the next PR after it is complete.
@souravzzz

Reply:
Ok

Reply:
@souravzzz please review and approve for merge to master 
This will bring master in sync with v1.8.x